### PR TITLE
[FIX]:UI Aligned in Pattern Configration

### LINKED ISSUE #4946: [UI] label not aligned with icon
#### Current Behavior
The label of the text field is not aligned with the icon beside it in the pattern configurator. 

#### Desired Behavior
they should be aligned. 

#### Screenshots/Logs
Screenshot 2022-01-15 at 12 45 44 PM

#### Environment

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Sonichigo**
**Description**

This PR fixes #4946 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by warunicorn19 — APPROVED** [review]
LGTM! Thank you @Sonichigo 
Screenshot 2022-01-25 at 11 31 48 PM

File: ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomInputField.js [hunk 1]
```diff
@@ -25,15 +25,15 @@ const CustomInputField = (props) => {
   return (
     <div key={props.id} style={style}>
       <TextField
-        InputLabelProps={{ style : { pointerEvents : "auto", padding : "3px" } }}
+        InputLabelProps={{ style : { pointerEvents : "auto", padding : "2px" } }}
```
Comment (leecalcote) [inline ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomInputField.js:28]: @Sonichigo thanks for this. Will you confirm equidistant spacing for the "?" and the placeholder text? Both are 2px from either left or right edge, is that right?

Comment (Sonichigo) [reply ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomInputField.js:28]: Right! both are 2px

**Review by Abhishek-kumar09 — APPROVED** [review]

File: ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomInputField.js (hunk 1 above)
Comment (leecalcote) [reply ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomInputField.js:28]: Excellent. 😄

**Comment by leecalcote** [comment]
yay, @Sonichigo! 🎈

**Pull request merged**